<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompatDB</class>
 <widget class="QWizard" name="CompatDB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>482</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>410</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Report Compatibility</string>
  </property>
  <property name="options">
   <set>QWizard::DisabledBackButtonOnLastPage|QWizard::HelpButtonOnRight|QWizard::NoBackButtonOnStartPage</set>
  </property>
  <widget class="QWizardPage" name="wizard_Info">
   <property name="title">
    <string>Report Game Compatibility</string>
   </property>
   <attribute name="pageId">
    <string notr="true">0</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lbl_Spiel">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Should you choose to submit a test case to the &lt;/span&gt;&lt;a href=&quot;https://yuzu-emu.org/game/&quot;&gt;&lt;span style=&quot; font-size:10pt; text-decoration: underline; color:#0000ff;&quot;&gt;yuzu Compatibility List&lt;/span&gt;&lt;/a&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;, The following information will be collected and displayed on the site:&lt;/span&gt;&lt;/p&gt;&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Hardware Information (CPU / GPU / Operating System)&lt;/li&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Which version of yuzu you are running&lt;/li&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;The connected yuzu account&lt;/li&gt;&lt;/ul&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizard_GameBoot">
   <property name="title">
    <string>Report Game Compatibility</string>
   </property>
   <attribute name="pageId">
    <string notr="true">1</string>
   </attribute>
   <layout class="QFormLayout" name="formLayout">
    <item row="2" column="0">
     <widget class="QRadioButton" name="radioButton_GameBoot_Yes">
      <property name="text">
       <string>Yes</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="lbl_GameBoot_Yes">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Game starts to output video or audio&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QRadioButton" name="radioButton_GameBoot_No">
      <property name="text">
       <string>No </string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="lbl_GameBoot_No">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Game doesn't get past the loading yuzu screen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="lbl_Independent">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Does the game boot?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
   <widget class="QWizardPage" name="wizard_GamePlay">
     <property name="title">
       <string>Report Game Compatibility</string>
     </property>
     <attribute name="pageId">
       <string notr="true">2</string>
     </attribute>
     <layout class="QFormLayout" name="formLayout">
       <item row="2" column="0">
         <widget class="QRadioButton" name="radioButton_Gameplay_Yes">
           <property name="text">
             <string>Yes</string>
           </property>
         </widget>
       </item>
       <item row="2" column="1">
         <widget class="QLabel" name="lbl_Gameplay_Yes">
           <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;You can get past the intro/menu and into gameplay&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="wordWrap">
             <bool>true</bool>
           </property>
         </widget>
       </item>
       <item row="4" column="0">
         <widget class="QRadioButton" name="radioButton_Gameplay_No">
           <property name="text">
             <string>No </string>
           </property>
         </widget>
       </item>
       <item row="4" column="1">
         <widget class="QLabel" name="lbl_Gameplay_No">
           <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The game crashes or freezes while loading or using the menu&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="wordWrap">
             <bool>true</bool>
           </property>
         </widget>
       </item>
       <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="lbl_Independent">
           <property name="font">
             <font>
               <pointsize>10</pointsize>
             </font>
           </property>
           <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Does the game reach gameplay?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="wordWrap">
             <bool>true</bool>
           </property>
         </widget>
       </item>
       <item row="1" column="0" colspan="2">
         <spacer name="verticalSpacer">
           <property name="orientation">
             <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
             <size>
               <width>20</width>
               <height>0</height>
             </size>
           </property>
         </spacer>
       </item>
     </layout>
   </widget>
   <widget class="QWizardPage" name="wizard_Freeze">
     <property name="title">
       <string>Report Game Compatibility</string>
     </property>
     <attribute name="pageId">
       <string notr="true">3</string>
     </attribute>
     <layout class="QFormLayout" name="formLayout">
       <item row="2" column="0">
         <widget class="QRadioButton" name="radioButton_Freeze_Yes">
           <property name="text">
             <string>Yes</string>
           </property>
         </widget>
       </item>
       <item row="2" column="1">
         <widget class="QLabel" name="lbl_Freeze_Yes">
           <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The game crashes or freezes while loading or using the menu&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="wordWrap">
             <bool>true</bool>
           </property>
         </widget>
       </item>
       <item row="4" column="0">
         <widget class="QRadioButton" name="radioButton_Freeze_No">
           <property name="text">
             <string>No </string>
           </property>
         </widget>
       </item>
       <item row="4" column="1">
         <widget class="QLabel" name="lbl_Freeze_No">
           <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The game never stops working&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="wordWrap">
             <bool>true</bool>
           </property>
         </widget>
       </item>
       <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="lbl_Independent">
           <property name="font">
             <font>
               <pointsize>10</pointsize>
             </font>
           </property>
           <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Does the game crash, freeze or lockup during gameplay?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="wordWrap">
             <bool>true</bool>
           </property>
         </widget>
       </item>
       <item row="1" column="0" colspan="2">
         <spacer name="verticalSpacer">
           <property name="orientation">
             <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
             <size>
               <width>20</width>
               <height>0</height>
             </size>
           </property>
         </spacer>
       </item>
     </layout>
   </widget>
   <widget class="QWizardPage" name="wizard_Freeze_optional">
     <property name="title">
       <string>Report Game Compatibility</string>
     </property>
     <attribute name="pageId">
       <string notr="true">4</string>
     </attribute>
     <layout class="QFormLayout" name="formLayout">
       <item row="2" column="0">
         <widget class="QCheckBox" name="checkbox_Freeze_crash">
           <property name="text">
             <string>The game crashes at specific points</string>
           </property>
         </widget>
       </item>
       <item row="3" column="0">
         <widget class="QCheckBox" name="checkbox_Freeze_random_crash">
           <property name="text">
             <string>The game randomly crashes</string>
           </property>
         </widget>
       </item>
       <item row="4" column="0">
         <widget class="QCheckBox" name="checkbox_Freeze_freeze">
           <property name="text">
             <string>The game freezes at specific points</string>
           </property>
         </widget>
       </item>
       <item row="5" column="0">
         <widget class="QCheckBox" name="checkbox_Freeze_random_freeze">
           <property name="text">
             <string>The game randomly freezes </string>
           </property>
         </widget>
       </item>
       <item row="6" column="0">
         <widget class="QCheckBox" name="checkbox_Freeze_softlock">
           <property name="text">
             <string>The game softlocks</string>
           </property>
         </widget>
       </item>
       <item row="7" column="0">
         <widget class="QCheckBox" name="checkbox_Freeze_memoryleak">
           <property name="text">
             <string>The has memory leaks</string>
           </property>
         </widget>
       </item>
       <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="lbl_Independent">
           <property name="font">
             <font>
               <pointsize>10</pointsize>
             </font>
           </property>
           <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Check the boxes with the issue&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="wordWrap">
             <bool>true</bool>
           </property>
         </widget>
       </item>
       <item row="1" column="0" colspan="2">
         <spacer name="verticalSpacer">
           <property name="orientation">
             <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
             <size>
               <width>20</width>
               <height>0</height>
             </size>
           </property>
         </spacer>
       </item>
     </layout>
   </widget>
   <widget class="QWizardPage" name="wizard_Graphical">
     <property name="title">
       <string>Report Game Compatibility</string>
     </property>
     <attribute name="pageId">
       <string notr="true">5</string>
     </attribute>
     <layout class="QFormLayout" name="formLayout">
       <item row="2" column="0">
         <widget class="QRadioButton" name="radioButton_Graphical_Yes">
           <property name="text">
             <string>Yes</string>
           </property>
         </widget>
       </item>
       <item row="2" column="1">
         <widget class="QLabel" name="lbl_Graphical_Yes">
           <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The game has graphical errors&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="wordWrap">
             <bool>true</bool>
           </property>
         </widget>
       </item>
       <item row="4" column="0">
         <widget class="QRadioButton" name="radioButton_Graphical_No">
           <property name="text">
             <string>No </string>
           </property>
         </widget>
       </item>
       <item row="4" column="1">
         <widget class="QLabel" name="lbl_Graphical_No">
           <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Everything is rendered as it looks on the Nintendo Switch&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="wordWrap">
             <bool>true</bool>
           </property>
         </widget>
       </item>
       <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="lbl_Independent">
           <property name="font">
             <font>
               <pointsize>10</pointsize>
             </font>
           </property>
           <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Does the game have any graphical glitches?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="wordWrap">
             <bool>true</bool>
           </property>
         </widget>
       </item>
       <item row="1" column="0" colspan="2">
         <spacer name="verticalSpacer">
           <property name="orientation">
             <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
             <size>
               <width>20</width>
               <height>0</height>
             </size>
           </property>
         </spacer>
       </item>
     </layout>
   </widget>
   <widget class="QWizardPage" name="wizard_Graphical_optional">
     <property name="title">
       <string>Report Game Compatibility</string>
     </property>
     <attribute name="pageId">
       <string notr="true">6</string>
     </attribute>
     <layout class="QFormLayout" name="formLayout">
       <item row="2" column="0">
         <widget class="QCheckBox" name="checkbox_Graphical_missing">
           <property name="text">
             <string>Missing textures</string>
           </property>
         </widget>
       </item>
       <item row="3" column="0">
         <widget class="QCheckBox" name="checkbox_Graphical_corrupted">
           <property name="text">
             <string>Corruped textures</string>
           </property>
         </widget>
       </item>
       <item row="4" column="0">
         <widget class="QCheckBox" name="checkbox_Graphical_fliquer">
           <property name="text">
             <string>Fliquering graphics</string>
           </property>
         </widget>
       </item>
       <item row="5" column="0">
         <widget class="QCheckBox" name="checkbox_Graphical_wrong_orientation">
           <property name="text">
             <string>Wrong orientation</string>
           </property>
         </widget>
       </item>
       <item row="6" column="0">
         <widget class="QCheckBox" name="checkbox_Graphical_wrong_colors">
           <property name="text">
             <string>Wrong colors</string>
           </property>
         </widget>
       </item>
       <item row="7" column="0">
         <widget class="QCheckBox" name="checkbox_Graphical_wrong_animation">
           <property name="text">
             <string>Wrong animation</string>
           </property>
         </widget>
       </item>
       <item row="8" column="0">
         <widget class="QCheckBox" name="checkbox_Graphical_overlay">
           <property name="text">
             <string>Overlayed images</string>
           </property>
         </widget>
       </item>
       <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="lbl_Independent">
           <property name="font">
             <font>
               <pointsize>10</pointsize>
             </font>
           </property>
           <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Check the boxes with the issue&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="wordWrap">
             <bool>true</bool>
           </property>
         </widget>
       </item>
       <item row="1" column="0" colspan="2">
         <spacer name="verticalSpacer">
           <property name="orientation">
             <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
             <size>
               <width>20</width>
               <height>0</height>
             </size>
           </property>
         </spacer>
       </item>
     </layout>
   </widget>
   <widget class="QWizardPage" name="wizard_Audio">
     <property name="title">
       <string>Report Game Compatibility</string>
     </property>
     <attribute name="pageId">
       <string notr="true">7</string>
     </attribute>
     <layout class="QFormLayout" name="formLayout">
       <item row="2" column="0">
         <widget class="QRadioButton" name="radioButton_Audio_Yes">
           <property name="text">
             <string>Yes</string>
           </property>
         </widget>
       </item>
       <item row="2" column="1">
         <widget class="QLabel" name="lbl_Audio_Yes">
           <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Audio is missing, faster, slower, corrupted, etc.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="wordWrap">
             <bool>true</bool>
           </property>
         </widget>
       </item>
       <item row="4" column="0">
         <widget class="QRadioButton" name="radioButton_Audio_No">
           <property name="text">
             <string>No </string>
           </property>
         </widget>
       </item>
       <item row="4" column="1">
         <widget class="QLabel" name="lbl_Audio_No">
           <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Audio is played perfectly, or almost perfectly with minimum distortion&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="wordWrap">
             <bool>true</bool>
           </property>
         </widget>
       </item>
       <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="lbl_Independent">
           <property name="font">
             <font>
               <pointsize>10</pointsize>
             </font>
           </property>
           <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Does the game have any audio glitches / missing effects?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="wordWrap">
             <bool>true</bool>
           </property>
         </widget>
       </item>
       <item row="1" column="0" colspan="2">
         <spacer name="verticalSpacer">
           <property name="orientation">
             <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
             <size>
               <width>20</width>
               <height>0</height>
             </size>
           </property>
         </spacer>
       </item>
     </layout>
   </widget>
   <widget class="QWizardPage" name="wizard_Audio_optional">
     <property name="title">
       <string>Report Game Compatibility</string>
     </property>
     <attribute name="pageId">
       <string notr="true">8</string>
     </attribute>
     <layout class="QFormLayout" name="formLayout">
       <item row="2" column="0">
         <widget class="QCheckBox" name="checkbox_Audio_faster">
           <property name="text">
             <string>Audio is faster</string>
           </property>
         </widget>
       </item>
       <item row="3" column="0">
         <widget class="QCheckBox" name="checkbox_Audio_slower">
           <property name="text">
             <string>Audio is slower</string>
           </property>
         </widget>
       </item>
       <item row="4" column="0">
         <widget class="QCheckBox" name="checkbox_Audio_loud">
           <property name="text">
             <string>Volume too high</string>
           </property>
         </widget>
       </item>
       <item row="5" column="0">
         <widget class="QCheckBox" name="checkbox_Audio_quiet">
           <property name="text">
             <string>Volume too low</string>
           </property>
         </widget>
       </item>
       <item row="6" column="0">
         <widget class="QCheckBox" name="checkbox_Freeze_Audio_corrupted">
           <property name="text">
             <string>Corrupted audio</string>
           </property>
         </widget>
       </item>
       <item row="7" column="0">
         <widget class="QCheckBox" name="checkbox_Freeze_Audio_missing">
           <property name="text">
             <string>Missing audio</string>
           </property>
         </widget>
       </item>
       <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="lbl_Independent">
           <property name="font">
             <font>
               <pointsize>10</pointsize>
             </font>
           </property>
           <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Check the boxes with the issue&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="wordWrap">
             <bool>true</bool>
           </property>
         </widget>
       </item>
       <item row="1" column="0" colspan="2">
         <spacer name="verticalSpacer">
           <property name="orientation">
             <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
             <size>
               <width>20</width>
               <height>0</height>
             </size>
           </property>
         </spacer>
       </item>
     </layout>
   </widget>
   <widget class="QWizardPage" name="wizard_Complete">
     <property name="title">
       <string>Report Game Compatibility</string>
     </property>
     <attribute name="pageId">
       <string notr="true">9</string>
     </attribute>
     <layout class="QFormLayout" name="formLayout">
       <item row="2" column="0">
         <widget class="QRadioButton" name="radioButton_Complete_Yes">
           <property name="text">
             <string>Yes</string>
           </property>
         </widget>
       </item>
       <item row="2" column="1">
         <widget class="QLabel" name="lbl_Complete_Yes">
           <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;You can finish the game without any workaround&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="wordWrap">
             <bool>true</bool>
           </property>
         </widget>
       </item>
       <item row="4" column="0">
         <widget class="QRadioButton" name="radioButton_Complete_No">
           <property name="text">
             <string>No </string>
           </property>
         </widget>
       </item>
       <item row="4" column="1">
         <widget class="QLabel" name="lbl_Complete_No">
           <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;You can't get past a certain area&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="wordWrap">
             <bool>true</bool>
           </property>
         </widget>
       </item>
       <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="lbl_Independent">
           <property name="font">
             <font>
               <pointsize>10</pointsize>
             </font>
           </property>
           <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Is the game completely playable from start to finish?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="wordWrap">
             <bool>true</bool>
           </property>
         </widget>
       </item>
       <item row="1" column="0" colspan="2">
         <spacer name="verticalSpacer">
           <property name="orientation">
             <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
             <size>
               <width>20</width>
               <height>0</height>
             </size>
           </property>
         </spacer>
       </item>
     </layout>
   </widget>
  <widget class="QWizardPage" name="wizard_ThankYou">
   <property name="title">
    <string>Thank you for your submission!</string>
   </property>
   <attribute name="pageId">
    <string notr="true">10</string>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
